fastly/resource_fastly_alert.go

```diff
@@ -0,0 +1,309 @@
+package fastly
+
+import (
+	"context"
+	"log"
+
+	gofastly "github.com/fastly/go-fastly/v9/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceFastlyAlert() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceFastlyAlertCreate,
+		ReadContext:   resourceFastlyAlertRead,
+		UpdateContext: resourceFastlyAlertUpdate,
+		DeleteContext: resourceFastlyAlertDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Text body included in Alert notifications.",
```

Comment:
This kinda read weird. Can we make these descriptions align (where possible) with the API docs?

For example...

```suggestion
				Description: "Additional text that is included in the alert notification.",
```

Reply:
Yes, done.